### PR TITLE
Feature/id 209 tab bar title

### PR DESCRIPTION
**Pull request by spinhalf-com**
## Purpose

_Briefly describe the purpose of the change._

Fixes ###-####

## Approach

_Explain how your code addresses the purpose of the change._

## Learning

_Any tips and tricks, blog posts or tools which helped you._

## Checklist

* [ x] I have performed a self-review of my own code
* [ x] I have added relevant logging with appropriate levels to my code
* [ x] I have updated documentation where relevant
* [ x] I have added tests to prove my work
* [ x] I have run an accessibility tool against the changes and applied fixes
* [ ] The team have tested these changes

### CODE REVIEW

File: service-front/module/Application/src/Controller/DonorFlowController.php [hunk 1]
```diff
@@ -38,6 +38,7 @@ public function howWillDonorConfirmAction(): ViewModel|Response
         $optionsdata = $this->config['opg_settings']['identity_methods'];
         $detailsData = $this->opgApiService->getDetailsData($uuid);
 
+        $view->setVariable('date_sub_form', $dateSubForm);
```
Comment (gregtyler) [inline service-front/module/Application/src/Controller/DonorFlowController.php:41]: Is this supposed to be here?

File: service-front/module/Application/src/Forms/PassportNumber.php [hunk 2]
```diff
@@ -8,6 +8,7 @@
 use Application\Validators\PassportInDateValidator;
 use Laminas\Form\Annotation;
 use Laminas\Hydrator\ObjectPropertyHydrator;
+use Laminas\Validator\NotEmpty;
```
Comment (gregtyler) [inline service-front/module/Application/src/Forms/PassportNumber.php:11]: Is this supposed to be here?

File: service-front/web/main.scss [hunk 3]
```diff
@@ -52,3 +52,7 @@ $govuk-assets-path: "../assets/";
 .table-header {
   background-color: $govuk-border-colour;
 }
+
+.app-link-text-color {
+  color: #1D70B8
+}
```
Comment (gregtyler) [inline service-front/web/main.scss:58]: Is this supposed to be here?

File: service-front/module/Application/view/layout/layout.twig [hunk 4]
```diff
@@ -3,7 +3,7 @@
 
 <head>
     <meta charset="utf-8">
-    <title>OPG Paper Identity</title>
+    <title>{% block title %}GOV.UK - The best place to find government services and information{% endblock %}</title>
```
Comment (gregtyler) [inline service-front/module/Application/view/layout/layout.twig:6]: We probably want the default header to be something relevant, like:
```suggestion
    <title>{% block title %}OPG Sirius - Identity check{% endblock %}</title>
```

File: service-front/module/Application/src/Controller/DonorFlowController.php (hunk 1 above)
Comment (spinhalf-com) [reply service-front/module/Application/src/Controller/DonorFlowController.php:41]: Yes - there's now a passport date check on the first page

**Review by gregtyler — APPROVED** [review]

**Pull request merged**